src/common/utils/scraper.ts

```diff
@@ -1,6 +1,6 @@
-import puppeteer, { Browser } from 'puppeteer';
-import prisma from '../../common/lib/prisma';
-import { TotoResult, TotoWinningPool } from '@prisma/client';
+import { Browser } from 'puppeteer';
+import prisma from '../lib/prisma';
+import { TotoResult } from '@prisma/client';
```

Comment:
import order seems wrong 🤔 . can we also use absolute import for `prisma`

Reply:
updated to absolute imports